docs/.vuepress/plugins/markdown-it-conditional-container/index.js

```diff
@@ -0,0 +1,142 @@
+const chalk = require('chalk');
+const { getEnvDocsFramework, parseFramework, getDefaultFramework, getNormalizedPath } = require('../../helpers');
+
+/* eslint-disable */
+/**
+ * Container used to display/hide blocks of content relevant to specific frameworks.
+ * It relies on providing the framework name(s) as arguments to the container (see 'Usage' below) and setting a
+ * `DOCS_FRAMEWORK` environmental variable as the framework name, for example:
+ *
+ * ```
+ * DOCS_FRAMEWORK=react
+ * ```
+ *
+ * Usage:
+ * ```
+ * ::: only-for react
+ * Content to be displayed only for React documentation.
+ * :::
+ *
+ * ::: only-for javascript react vue
+ * Content to be displayed only for JS, React and Vue documentation.
+ * :::
+ * ```
+ */
+module.exports = function conditionalContainer(markdown) {
+  const openAndCloseTagOneliner = /::: only-for (((react|javascript) ?)+)(.*?):::$/ms; // It is multi line text.
+  const openTokenContent = /(?:\n?)::: only-for (((react|javascript) ?)+)\n?/;
+  const fullMatchOpenToken = /^(?:\n?)::: only-for (((react|javascript) ?)+)\n?$/;
+  const closeTokenContent = /(?:\n?):::(?:\n?)$/;
+  const fullMatchCloseToken = /^(?:\n?):::(?:\n?)$/;
+  const markupForCustomContainer = ':::';
+  const newLineTokenType = 'softbreak';
+  const capturedGroupIndex = 1;
+  let endIndex;
+
+  const removeValueAndNewLine = ({ token, regexp, env }) => {
+    // Removing value from token's content.
+    token.content = token.content.replace(regexp, '');
+
+    // Some tags may don't have children.
+    if (token.children === null) {
+      return;
+    }
+
+    let childrenIndex = token.children.findIndex(childrenToken => regexp.test(childrenToken.content));
+
+    // Removing token's children also representing removed value.
```

Comment:
I hope that it will be more clear now. Changed within: https://github.com/handsontable/handsontable/pull/9627/commits/1ad35c495b33044153538560c899cbc376d3a741.